README.md

```diff
@@ -41,6 +41,11 @@ This repo is only available to Mozilla employees. If you have access to the rele
  - `third_party/svr/` for Snapdragon (should contain a `libs` folder, among other things)
  - `third_party/wavesdk/` for Vive (should contain a `build` folder, among other things)
 
+The [repo in `third_party`](https://github.com/MozillaReality/FirefoxReality-android-third-party) can be updated like so:
+
+```bash
+pushd third_party && git checkout master && git pull && popd
```

Comment:
I like your suggestion better. changed.

I use `git pull` which does `git pull --rebase` because I have `rebase = true` in my `~/.gitconfig`:

```ini
[branch "master"]
	remote = origin
	merge = refs/heads/master
	rebase = true
```